README.md

```diff
@@ -44,94 +98,211 @@ Open inbound ports on your security group. Use the table below as a guide. Make
-* All metadata reside in an Elasticsearch database. Make sure you have the domain name of that Elasticsearch instance handy.
-* Have the domain name of the _dos-dss server_ handy.
+Now you can begin installation. You will clone the repository on VM and
+run the bootstrap script. Be sure to set your branch to
+`feature/commons` as these instruction are specific to this branch.
+
+1. SSH onto your VM you created.
+
+1. clone this repo
+
+   ```
+   git clone https://github.com/DataBiosphere/cgp-deployment.git
+   ```
+1. Change the repo's directory
+   ```
+   cd cgp-deployment
+   ```
+
+1. Check out the correct branch
+   ```
+   git checkout feature/commons
+   ```
+
+1. Run the install script
+   ```
+   sudo bash install_bootstrap
+   ```
+
+The install script will ask for lots of information. Here we'll explain
+what to put for each step.
+
+1. First the Installer asks to install Docker and other dependencies.
+   These are necessary to continue.
+
+1. Next, the install will ask to launch the public-facing gateway nginx
+   server. You can do this in either `dev` mode or `prod` mode. For more
+   details see the [Installing in `prod` mode](#prod) and [Installing in
+   `dev` mode](#dev) sections.
+
+1. Now you will have to decide whether to launch boardwalk in `dev` or
+   `prod` modes.
+
+1. Next you are asked to decide if you want an authorization whitelist.
+
+   If you decide to use a whitelist you will need set it up with
+   [bouncer](https://github.com/ucsc-cgp/bouncer#how-to-use). You will
+   also be prompted for a project name and a contact email which will be
+   used in error messages for users who aren't authorized, but attempt
+   to access the data.
+
+1. You will now have to enter the hostname of dcc-dashboard. This is the
+   domain you made in Rounte 52, above. You need to know the name of the
+   domain, but at the time of installation the domain (or _record set_)
+   does not _actually_ have to be configured in _Route 53_
+
+1. Next you are prompted for AWS credentials. These can be made
+   specifically for the instance, or for a dev instance you could use
+   your own.
+
+1. The elasticsearch instance domain can be one that is prexisting if
+   used only for development.
+
+1. Next you will need a Google Client ID and Client secret. If you
+   missed it before, instructions to set this up are found on the
+   [Boardwalk deployment page](boardwalk/README.md#deployment).
+   _Boardwalk_ has functionality to export metadata to Broad's
+   FireCloud. In order to use it you need to provide your Google Cloud
+   Platform credentials.
+
+   You will also be asked for a Google site verfication code. For this
+   field you can just enter `NONE`.
+
+1. The bagit-firecloud-lambda is use for the **Export to Firecloud**
+   button.
+
+1. The [S3 bucket you created earlier](#makebucket) is what you should
+   use for this next step.
+
+1. The dos-dss server is needed next. Again you can use a prexisting one
```

Comment:
I think you will always use an existing one.

Reply:
I think you missed this comment.

Reply:
I'll link to https://github.com/DataBiosphere/cgp-data-store as we discussed for more context.